From: Robert Luberda <robert@debian.org>
Date: Sun, 6 Mar 2016 13:16:16 +0100
Subject: Remove references to fm/ dir and license.htm

Remove references to fm/* which describes Windows GUI,  which
is not available in this package.

Replace references to license.htm with references to copyright.

Drop .exe extension from 7z commands (LP: #181402).

Bugs-Ubuntu https://bugs.launchpad.net/ubuntu/+source/p7zip/+bug/181402
---
 DOC/MANUAL/cmdline/index.htm        | 10 +++++-----
 DOC/MANUAL/cmdline/switches/sfx.htm |  2 +-
 DOC/MANUAL/general/index.htm        |  3 +--
 DOC/MANUAL/start.htm                |  3 +--
 4 files changed, 8 insertions(+), 10 deletions(-)

diff --git a/DOC/MANUAL/cmdline/index.htm b/DOC/MANUAL/cmdline/index.htm
index c3515a3..129c479 100644
--- a/DOC/MANUAL/cmdline/index.htm
+++ b/DOC/MANUAL/cmdline/index.htm
@@ -11,12 +11,12 @@
 <H1>Command Line Version User's Guide</H1>
 
 
-<P>7z.exe is the command line version of 7-Zip. 7z.exe uses 7z.dll
-from the 7-Zip package. 7z.dll is used by the 7-Zip File Manager also.</LI>
+<P>7z is the command line version of 7-Zip. 7z uses 7z.so
+from the 7-Zip package.
 
-<P>7za.exe (a = alone) is a standalone version of 7-Zip.
-7za.exe supports only 7z, lzma, cab, zip, gzip, bzip2, Z and tar formats.
-7za.exe doesn't use external modules.</LI>
+<P>7za (a = alone) is a standalone version of 7-Zip.
+7za supports only 7z, lzma, cab, zip, gzip, bzip2, Z and tar formats.
+7za doesn't use external modules.
 
 <UL>
   <LI><A href = "syntax.htm">Command Line syntax</A></LI>
diff --git a/DOC/MANUAL/cmdline/switches/sfx.htm b/DOC/MANUAL/cmdline/switches/sfx.htm
index bed2f33..e93a284 100644
--- a/DOC/MANUAL/cmdline/switches/sfx.htm
+++ b/DOC/MANUAL/cmdline/switches/sfx.htm
@@ -22,7 +22,7 @@
   <DT><A name="SFX_Module"></A>{SFX_Module}</DT>
   <DD>
     <P>Specifies the SFX module that will be combined with the archive.
-       This module must be placed in the same directory as the 7z.exe.
+       This module must be placed in the same directory as the 7z.
        If {SFX_Module} is not assigned, 7-Zip will use standard console 
        SFX module 7zCon.sfx.</P>
     <TABLE>
diff --git a/DOC/MANUAL/general/index.htm b/DOC/MANUAL/general/index.htm
index 2e304a4..4c5e55a 100644
--- a/DOC/MANUAL/general/index.htm
+++ b/DOC/MANUAL/general/index.htm
@@ -13,7 +13,6 @@
 <h4>The main features of 7-Zip</h4>
 
 <UL>
-  <LI><A href = "../fm/index.htm">Powerful file manager</A></LI>
   <LI><A href = "performance.htm">High compression ratio and high speed</A></LI>
   <LI><A href = "formats.htm">Big number of supported archive formats</A></LI>
   <LI><A href = "../cmdline/index.htm">Additional command line version</A></LI>
@@ -21,7 +20,7 @@
 
 <H4>See Also</H4>
 <UL>
-  <LI><A href = "license.htm">License for use and distribution</A></LI>
+  <LI><A href = "../../../copyright">License for use and distribution</A></LI>
 </UL>
 
 </BODY>
diff --git a/DOC/MANUAL/start.htm b/DOC/MANUAL/start.htm
index e4f96b4..e767c49 100644
--- a/DOC/MANUAL/start.htm
+++ b/DOC/MANUAL/start.htm
@@ -17,7 +17,6 @@
 
 <UL>
   <LI><A href = "general/index.htm">General information about 7-Zip</A></LI>
-  <LI><A href = "fm/index.htm">User's Guide for 7-Zip File Manager</A></LI>
   <LI><A href = "cmdline/index.htm">User's Guide for command line version</A></LI>
 </UL>
 
@@ -25,7 +24,7 @@
 
 <UL>
   <LI><A href = "general/faq.htm">Frequently Asked Questions (FAQ)</A></LI>
-  <!-- <LI><A href = "general/license.htm">License for use and distribution</A></LI> -->
+  <!-- <LI><A href = "../../copyright">License for use and distribution</A></LI> -->
 </UL>
 
 <HR>
